packages/styleUtils/typography/stories/headingText.stories.tsx

```diff
@@ -41,14 +31,17 @@ const Template: Story<HeadingTextProps> = args => (
 );
 
 export const DefaultHeadingText1 = Template.bind({});
+DefaultHeadingText1.args = { align: "inherit" };
```

Comment:
TIL that `defaultValue` will be depreciated with v7. I've read the docs over some more, what we could do here instead to keep this drier is to pass `args` below the `argTypes` in the configuration above. 
Ex:
```
  args: {
    display: "inline",
  }
```
Now display inline will be applied to all of the stories so we don't have to pass it in individually.

Reply:
Going to have to revisit all of the defaultValue uses throughout in the near future, at least we have some time until v7 is stabilized. 😅

Reply:
Good spot! I actually read that in MIGRATION.md, but misinterpreted it 🤦‍♀️